internal/controller/storageclaim_controller.go

```diff
@@ -424,6 +469,44 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 				if err != nil {
 					return reconcile.Result{}, fmt.Errorf("failed to create or update VolumeSnapshotClass: %s", err)
 				}
+			case "VolumeReplicationClass":
```

Comment:
What happens when the drClusterConfig is deleted? Will the VRC's get deleted as well?